tests/test_statics.py

```diff
@@ -0,0 +1,29 @@
+import os
+
+from falcon.request import Request
+from falcon.response import Response
+from falcon.statics import Statics
+import falcon.testing as testing
+
+
+class MockStatics(Statics):
+
+    def _get_stream(self, path):
+        return path
+
+
+def test_serve_static_file():
+    static = MockStatics('/static', '/var/www/statics')
+
+    req = Request(testing.create_environ(
+        host='test.com',
+        path='/static/../css//test.css',
```

Comment:
We'll want to try lots of crazy paths to ensure a malicious client can't break out. You can use ```pytest.mark.parametrize``` to make tests like this a little easier.